scripts/indexes.py

```diff
@@ -8,63 +8,61 @@
 
 import argparse, sys, os, yaml
 
+parser = argparse.ArgumentParser(description='Generates custom index statements')
+parser.add_argument('--concurrent', dest='concurrent', help='Generate indexes CONCURRENTLY', action='store_true', default=False)
+parser.add_argument('--fillfactor', help='Custom fillfactor to use')
+parser.add_argument('--notexist', help='Use IF NOT EXISTS (requires 9.5)', action='store_true', default=False)
+parser.add_argument('--osm2pgsql', help='Include indexes normally built by osm2pgsql', action='store_true', default=False)
+parser.add_argument('--reindex', help='Rebuild existing indexes', action='store_true', default=False)
+parser.add_argument('--zero', '-0', help='Emit zero-separated index statements for use with xargs', action='store_true', default=False)
```

Comment:
I am not to familiar with xargs, so maybe it is an xarg thing, but the line 'zero-separated index statements' is rather cryptic to me. What does it mean? Maybe spell it out a bit more?

Reply:
I've renamed this `--null` like xargs, and reused language from its man page.